include/intx/int128.hpp

```diff
@@ -63,46 +63,56 @@ struct uint<128>
 {
     static constexpr unsigned num_bits = 128;
 
-    uint64_t lo = 0;
-    uint64_t hi = 0;
+    uint64_t words[2]{};
 
     constexpr uint() noexcept = default;
 
-    constexpr uint(uint64_t high, uint64_t low) noexcept : lo{low}, hi{high} {}
+    constexpr uint(uint64_t hi, uint64_t lo) noexcept : words{lo, hi} {}
```

Comment:
A bit of inconsistency, the input order is `hi`/`lo` but is stored "reversed" in `words`. Would it make sense (maybe not in this PR) to have this match?